google/cloud/firestore_v1/base_query.py

```diff
@@ -396,6 +396,18 @@ def limit(self, count: int) -> "BaseQuery":
             all_descendants=self._all_descendants,
         )
 
+    @property
+    def limit_type(self) -> int:
+        """BundledQuery.LimitType equivalent of this query.
```

Comment:
This seems a bit strange to me to end up in the query class, like the direction of dependence is almost reversed?

Reply:
Hmm, yes I think you're right here. I had thought of this as limits obviously being generic to queries, but these limits are specifically namespaced within the BundledQuery proto.

Will reverse.